src/app/clusters/time-synchronization-server/time-synchronization-server.cpp

```diff
@@ -144,10 +144,10 @@ static bool emitDSTTableEmptyEvent(EndpointId ep)
 
     if (CHIP_NO_ERROR != error)
     {
-        ChipLogError(Zcl, "Unable to emit DSTTableEmpty event [ep=%d]", ep);
+        ChipLogError(Zcl, "DSTTableEmptyEvent failed");
```

Comment:
We should probably log error here as well for debugging

Reply:
Agree. Done.